corgi/web/templates/product_manifest.json

```diff
@@ -1,4 +1,11 @@
 {# ProductModel manifest, obj is a ProductModel subclass here #}{% extends "base_manifest.json" %}{% load base_extras %}
+{% block document %}
+  "documentDescribes": [
+    "SPDXRef-{{external_name}}"
```

Comment:
As mentioned above, using the `obj.uuid` is really an option any more as multiple streams can now share an SBOM.

Reply:
I changed this to generate a `document uuid` for each generation of the manifest file and using it in the documentDescribes and package references. Because of the same_content check in the cpu_update_ps_manifest task this will now be updated whenever the stream manifest content is updated.

Reply:
I think it's more elegant to use a document-uuid instead of the stream uuid as the document can describe multiple streams, and might switch between them. This also means the document-uuid will be updated each time is updated, which is a good idea since we are really publishing the manifest of the latest builds in that stream.